### ISSUE TITLE
Weird permission denied error in mutli-user setup

### ISSUE THREAD

**Issue by frol** [conda/conda#3753]
Here is the error I am getting every second build:

```
Runtime error: Could not open '/opt/conda/pkgs/qq-1-0.tar.bz2.part' for writing ([Errno 13] Permission denied: '/opt/conda/pkgs/qq-1-0.tar.bz2.part').
```

I am using a multi-user setup where `/opt/conda` is not write-accessible by the regular user, which I am using to build a package.

Here is the full log:

```
$ conda build /tmp/conda.recipe/
BUILD START: qq-1-0
Copying /tmp/conda.recipe to /tmp/conda-bld/conda.recipe_1477655114859/work
Package: qq-1-0
source tree in: /tmp/conda-bld/conda.recipe_1477655114859/work
number of files: 0
Fixing permissions
Fixing permissions
updating index in: /tmp/conda-bld/linux-64
updating: qq-1-0.tar.bz2
TEST START: qq-1-0
updating index in: /tmp/conda-bld/linux-64
updating index in: /tmp/conda-bld/noarch

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    qq-1                       |                0           1 KB  local

The following NEW packages will be INSTALLED:

    qq: 1-0 local (copy)

WARNING conda.lock:touch(53): Failed to create lock, do not run conda in parallel processes [errno 13]
Runtime error: Could not open '/opt/conda/pkgs/qq-1-0.tar.bz2.part' for writing ([Errno 13] Permission denied: '/opt/conda/pkgs/qq-1-0.tar.bz2.part').
```

This error happens every second `conda build` run! Every first is successful!

Here is the minimal meta.yaml:

``` yaml
package:
  name: qq
  version: 1

source:
  path: .

test:
  commands:
    - ls
```

Conda info:

```
               platform : linux-64
          conda version : 4.2.11
       conda is private : False
      conda-env version : 4.2.11
    conda-build version : 2.0.7
         python version : 3.5.2.final.0
       requests version : 2.11.1
       root environment : /opt/conda  (read only)
    default environment : /tmp/.conda/envs/_root
       envs directories : /tmp/.conda/envs
                          /opt/conda/envs
          package cache : /tmp/.conda/envs/.pkgs
                          /opt/conda/pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : None
           offline mode : False
```

**Comment by bkreider** [comment]
Has this been fixed in newer versions of conda?  Any idea what version fixed it if it has been fixed?

**Comment by kalefranz** [comment]
To my knowledge, all multi-user permissions issues have been fixed in conda >= 4.3.13.
